app/recordtransfer/context_processors.py

```diff
@@ -16,9 +16,10 @@ def file_upload_status(request: HttpRequest) -> dict[str, Any]:
     return {"FILE_UPLOAD_ENABLED": settings.FILE_UPLOAD_ENABLED}
 
 
-def constants_context(request: HttpRequest) -> dict[str, str]:
+def constants_context(request: HttpRequest) -> dict[str, Any]:
     """Make constants available globally in all templates."""
     return {
         **constants.HtmlIds().asdict(),
         **constants.QueryParameters().asdict(),
+        "ACCEPTED_FILE_FORMATS": settings.ACCEPTED_FILE_FORMATS,
     }
```

Comment:
This context processor is for constants that are needed globally. If you look in `recordtransfer.views.pre_submission.SubmissionFormWizard._get_template_context`, we're already including some of the upload settings there for the upload files page. Can you add ACCEPTED_FILE_FORMATS there instead?

e.g.,

```python
        elif self.current_step == SubmissionStep.UPLOAD_FILES:
            context.update(
                {
                    "MAX_TOTAL_UPLOAD_SIZE_MB": settings.MAX_TOTAL_UPLOAD_SIZE_MB,
                    "MAX_SINGLE_UPLOAD_SIZE_MB": settings.MAX_SINGLE_UPLOAD_SIZE_MB,
                    "MAX_TOTAL_UPLOAD_COUNT": settings.MAX_TOTAL_UPLOAD_COUNT,
                    "ACCEPTED_FILE_FORMATS": settings.ACCEPTED_FILE_FORMATS,
                }
            )
```

Reply:
All good!